docker-compose.yaml

```diff
@@ -1,34 +1,84 @@
 
-version: "3.8"
-
+version: "3"
 services:
-  # account:
-  #   build: "src/account/"
-  #   command: "npm run account"
-  #   ports: 
-  #     - "5000:5000" 
-  #   env_file: "src/account/.env"
-  #   dns:
-  #     - "8.8.8.8"
-  models:
-    build: "src/models/"
-    command: "echo Models Container Working"
-  routes:
-    build: "src/routes/"
-    command: "npx ts-node github.ts"
+  activemq: 
+    image: rmohr/activemq
+    container_name: activemq
+    networks:
+      - backend
+    ports: 
+      - "8161:8161"
+      - "1883:1883"
+    env_file: ".env"
+
+  # server:

```

Comment:
@KevinLemon112 There was a 404 error in all of the user routes because they were not run on the docker compose server.  You will need to either uncomment this service or make a new user service to get the user routes to work.